lambda_function.py

```diff
@@ -1,14 +1,14 @@
 import json
 import requests
 import os
-import urllib.parse
 
 
 def lambda_handler(event, context):
     # Base URL of Gupshup's OTP API, including all constant params
-    baseURL = os.environ.get("OTP_API_BASE_URL")
+    base_URL = os.environ.get("OTP_API_BASE_URL")
```

Comment:
how is this working?
You've changed the name of that variable from `OTP_API_BASE_URL` to `API_BASE_URL` no?
Did you test it?